osi_sensorviewconfiguration.proto

```diff
@@ -913,3 +842,139 @@ message UltrasonicSensorViewConfiguration
+    // must be no value, indicating that the simulation environment
+    // cannot provide raytracer data in one of the requested formats.
+    //
+    optional RaytracerFormat raytracer_format = 9;
+
+    // Different predefined raytracer formats
+    // 
+    enum RaytracerFormat
+    {
+        // Type of channel format is unknown (must not be used).
+        //
+        RAYTRACER_FORMAT_UNKNOWN = 0;
+
+        // Unspecified but known channel format.
+        // Consider proposing an additional format if using
+        // \c #RAYTRACER_FORMAT_OTHER.
+        //
+        RAYTRACER_FORMAT_OTHER = 1;
+
+        // Radar raytracer format used in the Japanese DIVP project.
+        //
+        // The message consists of the following fields in the described order.
+        // In brackets the data type, the unit and a short description is given. 
+        // distance (float; in m; path length of the ray)
+        // relative_speed (float; in m/s; summed relative speed due to interaction points with moved surfaces)
+        // propagation_attenuation_h_pol (float; in W; attenuation of signal strength of horizontal polarisation)
+        // propagation_attenuation_v_pol (float; in W; attenuation of signal strength of vertical polarisation)
+        // direction_of_arrival_azimuth (float; in rad; direction of arrival of the ray in the configured raytracer coordinate system in azimuth direction)
+        // direction_of_arrival_elevation (float; in rad; direction of arrival of the ray in the configured raytracer coordinate system in elevation direction)
+        // direction_of_departure_azimuth (float; in rad; direction of departure of the ray in the configured raytracer coordinate system in azimuth direction)
+        // direction_of_departure_elevation (float; in rad; direction of departure of the ray in the configured raytracer coordinate system in elevation direction)
+        //
+        RAYTRACER_FORMAT_DIVP_RADAR = 2;
+        
+        // Radar raytracer format used in the German VIVALDI project.
+        //
+        // The message consists of the following fields in the described order.
+        // In brackets the data type, the unit and a short description is given. 
+        // intersection_path_length (float; in m; path length of the ray between the First_Hitpoint and the Last_Hitpoint)
+        // relative_speed (float; in m/s; summed relative speed due to interaction points with moved surfaces)
+        // jones_vector (float4; in V/m; jones vector of the ray with information of the electromagnetic wave`s phase, signal strength and polarisation)
+        // last_hitpoint (float3; in m; coordinates of the last hitpoint in the raytracer's coordinate system defined by the mounting position)
+        // first_hitpoint (float3; in m; coordinates of the first hitpoint in the raytracer's coordinate system defined by the mounting position)
+        //
+        RAYTRACER_FORMAT_VIVALDI_RADAR = 3;
+    }
```

Comment:
I am wondering why there is no `object_id` given per ray, has this been discussed?

Reply:
Yes, we've discussed this. An object_id would be only an debug interface for the ray tracing algorithm. Therefore we decided to remove it from the message.

Reply:
ok fine. We can still add it later, if needed.